### ISSUE TITLE
Speaker job title & company in bio

### ISSUE THREAD

**Issue by hlashbrooke** [WordPress/Learn#232]
The workshops speaker bios are currently pulled straight from their WordPress.org profiles. I was wondering if it would be possible to also pull the job title and description from their profiles too? Then we can display that long side their name or above their bio somewhere.

**Comment by jonathanbossenger** [comment]
@hlashbrooke I assume you mean the Presenters section at the bottom of the workshop transcript. Do you have any specific requests on how it should display that data?

**Comment by jonathanbossenger** [comment]
@coreymckrill I notice that in the `wporg_get_workshop_presenter_bio` function, it will only pull from usermeta for local environments. Can you possibly send me the meta keys for the two fields @hlashbrooke wants to add here, so I can set up my local usermeta correctly?

**Comment by hlashbrooke** [comment]
Yeah - this would be in the "presenters" section. That section was only given that title after I posted this issue.

For the display I was thinking something really simple like this:
![Screenshot 2021-10-24 08 03 54](https://user-images.githubusercontent.com/1543143/138568443-2a1aaeda-2e4b-494f-b3b8-f9343e25ed70.png)

I mocked that up in the Inspector, so it's not super precise, but that's what I was envisioning here. Other option would be below the avatar.

**Comment by jonathanbossenger** [comment]
Perfect, thanks @hlashbrooke

**Comment by jonathanbossenger** [comment]
@coreymckrill follow up question, what is the `$xprofile_field_id` for the job title and company name fields? Thanks

**Comment by jonathanbossenger** [comment]
@tellyworth I believe Corey is on sabbatical. I'd appreciate your help on my questions for this issue , thanks.

To summarise:

- I notice that in the wporg_get_workshop_presenter_bio function, it will only pull from usermeta for local environments. Can you possibly send me the meta keys for the two fields we want to add here, so I can set up my local usermeta correctly?
- additionally, what is the $xprofile_field_id for the job title and company name fields?

**Comment by jonathanbossenger** [comment]
I've opened a WIP PR in the hopes that it will still count towards Hacktoberfest once closed, but I'd like to get the `usermeta` keys and the profile_field_ids for the job title and company fields, in order to test this properly.
